ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>8.0</string>
+  <string>9.0</string>
```

Comment:
Why the iOS minimum version was also bumped?

Reply:
my bad, it automatically bumped when I updated flutter to the beta version (don't know exactly why, seems related with https://github.com/flutter/flutter/issues/58200)

Reply:
I don't know why Flutter is automatically bumping this version to 9 every time I build the iOS app:

![CleanShot 2021-07-28 at 17 15 56](https://user-images.githubusercontent.com/11745745/127389829-11336981-3d02-4f84-9280-da04d0cfb711.png)

There's nothing about the minimum version change in the [Breaking Changes](https://flutter.dev/docs/release/breaking-changes) guide, although they state in [Build and release an iOS app
](https://flutter.dev/docs/deployment/ios#review-xcode-project-settings) that the minimum supported version is iOS 9.0 (we also have deployment target set to 9.0 in the PBX)

![CleanShot 2021-07-28 at 17 19 07](https://user-images.githubusercontent.com/11745745/127390077-ddc83c0b-2e89-4cab-b8ac-254db14675e0.png)
![CleanShot 2021-07-28 at 17 18 58](https://user-images.githubusercontent.com/11745745/127390079-8e319723-b501-40d9-9505-528a20bdce06.png)

TLDR: I don't know why Flutter automatically bumped, but the version 9 seems to be the correct one